packages/atomic/src/components/commerce/product-template-components/atomic-product-image/atomic-product-image.tsx

```diff
@@ -0,0 +1,238 @@
+import {Product, ProductTemplatesHelpers} from '@coveo/headless/commerce';
```

Comment:
The PR needs an update from master/resolve merge conflict before we can review it.

Reply:
should be good now